p3/plot/backend/textemplates/navchart.tex

```diff
@@ -0,0 +1,95 @@
+\documentclass{standalone}
+\usepackage[dvipsnames]{xcolor}
```

Comment:
For the arrows on the navchart I use some colours not in the base pgfplots class (ForestGreen, for example). It could be removed by specifying the colors directly.